src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -22,6 +23,8 @@ using OptionalGenerationConfig = std::optional<GenerationConfig>;
 using EncodedInputs = std::variant<ov::Tensor, TokenizedInputs>;
 using StringInputs = std::variant<std::string, std::vector<std::string>>;
 
+static constexpr ov::Property<SchedulerConfig> scheduler_config{"scheduler_config"};
```

Comment:
Is it actually needed? `Property` was added to enable positional arguments. But `ov::AnyMap` is used to pass `SchedulerConfig` now. I suspect `AnyMap` should be able to store `SchedulerConfig` without `ov::Property` definition.

Reply:
Never mind